test/self_protocore/upsert_validator.js

```diff
@@ -107,7 +107,7 @@ contract('SelfProtocore::upsertValidator', (accounts) => {
       );
     });
 
-    it('should remove validator if reputation is 0', async () => {
+    it.skip('should remove validator if reputation is 0', async () => {
```

Comment:
Why the positive test case is skipped ?

Reply:
@jayeshbairagi, please note this down to create a followup ticket.

Reply:
Noted